### PR TITLE
Add actions-display-video-360

### PR DESCRIPTION
**Pull request by marinhero**
Just scaffolding a new destination. Metadata and more to come.

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/src/destinations/display-video-360/index.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import addToAudience from './addToAudience'
+
+import removeFromAudience from './removeFromAudience'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Display Video 360',
```
[the remark this answers is not in the document]
Comment (nick-Ag) [reply packages/destination-actions/src/destinations/display-video-360/index.ts:9]: Also very nit: Display & Video? Should there be an "&"?

**Review by nick-Ag — APPROVED** [review]

File: packages/destination-actions/src/destinations/display-video-360/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (marinhero) [reply packages/destination-actions/src/destinations/display-video-360/index.ts:9]: Uhm, good question. They will briefly overlap in the catalogue so I think the answer is Yes. Let my do that.

**Comment by marinhero** [comment]
All right. This is ready to be merged during the next actions deploy. It will require a registration and a push. Tagging @joe-ayoub-segment to get this under his radar. Thank you!

**Comment by joe-ayoub-segment** [comment]
hi @marinhero this PR has been deployed

**Pull request merged**